specification/compute/resource-manager/Microsoft.Compute/stable/2020-12-01/disk.json

```diff
@@ -2026,7 +2067,7 @@
           }
         },
         "purchasePlan": {
-          "$ref": "#/definitions/PurchasePlan",
+          "$ref": "#/definitions/DiskPurchasePlan",
```

Comment:
why the name change? should we have a SnapshotPurchasePlan?

Reply:
PurchasePlan definition already exists for other compute resources, we just needed a different defnition name, thus DiskPurchasePlan.  PurchasePlan for snapshot uses the same defintion so no need to add SnapshotPurchasePlan.